mutacc/builds/build_variant.py

```diff
@@ -18,19 +18,40 @@ class Variant(dict):
         Class to represent variant
     """
 
-    def __init__(self, vcf_entry, samples, padding, parser=None):
+    def __init__(self, vcf_entry, samples, padding, sv_padding, parser=None):
 
         super(Variant, self).__init__()
 
         self.entry = vcf_entry
         self.samples = samples
-
-        self.build_variant_object(padding)
-        self.entry = str(self.entry)
+        self.padding = padding
+        self.sv_padding = sv_padding
+        self.build_variant_object()
 
         if parser is not None:
             self.update(parser.parse(vcf_entry))
 
+        del self.entry
+
+    def _get_padding(self, variant_type):
+
+        if variant_type == "SNV":
+            return self.padding
+
+        if self.entry.INFO.get("CIPOS") or self.entry.INFO.get("CIEND"):
+
+            cis = []
```

Comment:
What is cis in this context?

Reply:
confidence intervals 😄 will rename variable to something nicer.